source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,266 @@
+term "Fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin caseSensitiveExample
+   :end-before: end caseSensitiveExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] }
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the phrase with escaped quotes to the ``Filters.text()`` method to
+search for a phrase in your text search. If you don't add escaped quotes,
+the method runs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+phrase "Fate of the Furious". 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+ 
+Search Text with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefix each term you want to exclude in your text search with a minus
+sign ``-``. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "Furious", but do not contain the term "Fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin negateExample
+   :end-before: end negateExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+.. _sort_by_relevance:
+
+Sort by Relevance
+~~~~~~~~~~~~~~~~~
+
+The text search assigns a numerical text score to indicate how relevant
+each result matches the search string. Often, you want to know which
+results are the most relevant.  
+
+To sort by relevance, you use the :doc:`Projections
+</fundamentals/builders/projections>`  and :doc:`Sorts
+</fundamentals/builders/sort>` builders.
+
+First, create a project document to retrieve a text score by using the
+``Projections.metaTextScore()`` method. Upon passing this document to the
+``projection()`` method generates a new field with the specified name
+and returns the documents text score. 
+
+Then, specify a sort criteria to sort by the text score by using the
+``Sorts.metaTextScore()`` method. Upon passing this to the ``sort()``
+method, MongoDB sorts the results from the most to least relevant. By
+default, the method returns the results unsorted. 
```

Comment:
It would be helpful to mention what you're calling the sort() method on.

Reply:
I meant that you call sort() on an instance of a MongoCollection. Since the reader might not know much about sort() yet as they're on the Text Search page, we should make this clear in a location that a reader might expect it such as when it's first introduced.